_styles.js

```diff
@@ -337,3 +337,40 @@ export const SaveButton = styled.button`
     background-color: darkblue;
   }
 `;
+
+export const FilterContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  overflow: hidden;
+
+  position: relative;
+  align-self: stretch;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  justify-content: center;
+  gap: 10px;
+  padding: 32px;
+`;
+
+export const FilterButton = styled.button`
+  padding: 10px 20px;
+  border: none;
+  border-radius: 20px;
+  cursor: pointer;
+  background-color: ${(props) => (props.active ? props.color : "#f0f0f0")};
+  color: ${(props) => (props.active ? "#fff" : "#333")};
```

Comment:
you should prefix props being passed to a styled component with "$". 
Otherwise you will get this message in the terminal:

> styled-components: it looks like an unknown prop "active" is being sent through to the DOM, which will likely trigger a React console error. If you would like automatic filtering of unknown props, you can opt-into that behavior via `<StyleSheetManager shouldForwardProp={...}>` (connect an API like `@emotion/is-prop-valid`) or consider using transient props (`$` prefix for automatic filtering.)
Warning: Received `false` for a non-boolean attribute `active`.

```suggestion
  background-color: ${(props) => (props.active ? props.$color : "#f0f0f0")};
  color: ${(props) => (props.$active ? "#fff" : "#333")};
```

I also think that "#fff" is not the best color, since in combination with some flavor colors, it is really hard to read.